stubs/cv2/__init__.pyi

```diff
@@ -0,0 +1,45 @@
+"""
+Type stubs for cv2.
+Note that stubs are only written for the parts that we use.
+"""
+from typing import List, Optional, Tuple, Union
+
+from numpy import array, ndarray
+
+CAP_PROP_BUFFERSIZE: int
+
+FILE_STORAGE_READ: int
+FILE_STORAGE_WRITE: int
+
+class aruco_DetectorParameters:
```

Comment:
Should this definition not live under the `aruco` module?

Reply:
```
>>> import cv2
>>> cv2.aruco.DetectorParameters
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'cv2.aruco' has no attribute 'DetectorParameters'
>>> cv2.aruco_DetectorParameters
<class 'cv2.aruco_DetectorParameters'>
```

Reply:
That's, odd.